client/scss/components/_listing.scss

```diff
@@ -487,6 +487,10 @@ table.listing {
       font-size: 0.8em;
     }
 
+    .locked-indicator-owner {
+      color: #f57c00;
+    }
+
```

Comment:
here is the new golden lock code

Reply:
We will also need to set up a variable for this colour - maybe review the existing colours for something suitable?
E.g. color-teal-darker?